protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -1128,8 +1232,9 @@ message IntervalToCandleId {
     string candle_id=2;
 }
 
-// A list of interval to candle id mappings for a given market
+// A list of mappings interval to candle ID for a given market
```

Comment:
Don't quite understand this one

Reply:
It means a list of mapped pairs "interval" -> "candle ID"